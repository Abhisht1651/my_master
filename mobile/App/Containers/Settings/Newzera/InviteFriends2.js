// @flow

import React from 'react'
import { Text, View, Image, Dimensions, TouchableOpacity } from 'react-native'
import { NavigationContainerProps } from '@react-navigation/native'

import { Images } from '../../../Themes'

import styles from './Styles/InviteFriendsStyle'
// import common from './Styles/CommonStyle'

const Width = Dimensions.get('window').width
const Height = Dimensions.get('window').height

type Props = NavigationContainerProps & {
  connections: number,
  emailName: Array<string>,
  emailPhotos: Array<string>,
  contactName: Array<string>,
  contactPhotos: Array<string>
}

export default function InviteFriends2(props: Props) {
  const { navigation } = props
  const { emailName, emailPhotos, connections } = props
  const { contactName, contactPhotos } = props

  const allListEmail = emailName.map((name, index) => (
    <View key={index} style={styles.flexRowInvite}>
      <View style={{ width: (105 / 375) * Width }}>
        <View style={{ paddingTop: (10 / 812) * Height }} />
        <View style={{ paddingLeft: (23 / 375) * Width }}>
          <Image source={emailPhotos[index]} key={index} style={styles.imagePerson} />
        </View>
        <View style={{ paddingTop: (11 / 812) * Height }} />
        <View style={{ paddingLeft: (20 / 375) * Width }}>
          <View style={{ width: (66 / 375) * Width }}>
            <Text style={styles.namePerson}>{name}</Text>
          </View>
        </View>
        <View style={{ paddingTop: (9 / 812) * Height }} />
        <View style={{ paddingLeft: (15 / 375) * Width }}>
          <View style={styles.inviteButton}>
            <View style={{ paddingTop: (5 / 812) * Height }}>
              <Text style={styles.invitePersonText}>Invite</Text>
            </View>
          </View>
        </View>
        <View style={{ paddingTop: (10 / 812) * Height }} />
      </View>
      <View style={{ paddingLeft: (5 / 375) * Width }}>
        <View style={styles.verticleLine} />
      </View>
    </View>
  ))

  const allListContact = contactName.map((name, index) => (
    <View key={index} style={styles.flexRowInvite}>
      <View style={{ width: (105 / 375) * Width }}>
        <View style={{ paddingTop: (10 / 812) * Height }} />
        <View style={{ paddingLeft: (23 / 375) * Width }}>
          <Image source={contactPhotos[index]} key={index} style={styles.imagePerson} />
        </View>
        <View style={{ paddingTop: (11 / 812) * Height }} />
        <View style={{ paddingLeft: (20 / 375) * Width }}>
          <View style={{ width: (66 / 375) * Width }}>
            <Text style={styles.namePerson}>{name}</Text>
          </View>
        </View>
        <View style={{ paddingTop: (9 / 812) * Height }} />
        <View style={{ paddingLeft: (15 / 375) * Width }}>
          <View style={styles.inviteButton}>
            <View style={{ paddingTop: (5 / 812) * Height }}>
              <Text style={styles.invitePersonText}>Invite</Text>
            </View>
          </View>
        </View>
        <View style={{ paddingTop: (10 / 812) * Height }} />
      </View>
      <View style={{ paddingLeft: (5 / 375) * Width }}>
        <View style={styles.verticleLine} />
      </View>
    </View>
  ))

  return (
    <View>
      <View style={{ paddingTop: (56 / 812) * Height }}>
        <View style={styles.flexRow}>
          <View style={{ paddingTop: (6 / 812) * Height }}>
            <TouchableOpacity onPress={() => navigation.navigate('NewzeraSettings')}>
              <Image source={Images.leftArrowback} style={styles.leftArrow} />
            </TouchableOpacity>
          </View>
          <View style={{ paddingLeft: (83 / 375) * Width }}>
            <Text style={styles.heading}>Invite Friends</Text>
          </View>
        </View>
        <View style={{ paddingTop: (31 / 812) * Height }}>
          <View>
            <View style={styles.line} />
          </View>
        </View>
        <View style={{ paddingTop: (33 / 812) * Height }} />
        <View style={{ paddingLeft: (25 / 375) * Width }}>
          <View style={{ width: (325 / 375) * Width }}>
            <Text style={styles.connect}>Weâ€™ve found {connections} connections</Text>
          </View>
        </View>
        <View style={{ paddingTop: (31 / 812) * Height }} />
        <View style={{ paddingLeft: (25 / 375) * Width }}>
          <View style={{ width: (325 / 375) * Width }}>
            <View style={styles.inviteHead}>
              <View>
                <Text style={styles.inviteText}>Invite friends via email</Text>
              </View>
              <View style={styles.inviteAllPos}>
                <Text style={styles.inviteAll}>Invite All</Text>
              </View>
            </View>
          </View>
          <View style={{ paddingTop: (31 / 812) * Height }} />
          <View style={styles.flexRowInvite}>{allListEmail}</View>
        </View>
        <View style={{ paddingTop: (51 / 812) * Height }} />
        <View style={{ paddingLeft: (25 / 375) * Width }}>
          <View style={{ width: (325 / 375) * Width }}>
            <View style={styles.inviteHead}>
              <View>
                <Text style={styles.inviteText}>Invite friends via contact</Text>
              </View>
              <View style={styles.inviteAllPos}>
                <TouchableOpacity onPress={() => navigation.navigate('InviteFriends1')}>
                  <Text style={styles.inviteAll}>Invite All</Text>
                </TouchableOpacity>
              </View>
            </View>
          </View>
          <View style={{ paddingTop: (31 / 812) * Height }} />
          <View style={styles.flexRowInvite}>{allListContact}</View>
        </View>
      </View>
    </View>
  )
}

InviteFriends2.defaultProps = {
  connections: 118,
  emailName: ['Amelia Jones', 'Amelia Jones', 'Amelia Jones'],
  emailPhotos: [Images.sean, Images.amanda, Images.trump],
  contactName: ['Name 1 ', 'Name 2', 'Name 3'],
  contactPhotos: [Images.sean, Images.amanda, Images.trump]
}
