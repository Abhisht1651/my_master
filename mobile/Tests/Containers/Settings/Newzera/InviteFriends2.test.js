// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import InviteFriends2 from '../../../../App/Containers/Settings/Newzera/InviteFriends2'

describe('InviteFriends2', () => {
  it('InviteFriends2 component renders correctly', () => {
    const tree = shallow(<InviteFriends2 />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the Text Component', () => {
    const wrapper = shallow(<InviteFriends2 />)
    expect(wrapper.find('Text')).toHaveLength(18)
  })

  it('should render the Image Component', () => {
    const wrapper = shallow(<InviteFriends2 />)
    expect(wrapper.find('Image')).toHaveLength(7)
  })
})
