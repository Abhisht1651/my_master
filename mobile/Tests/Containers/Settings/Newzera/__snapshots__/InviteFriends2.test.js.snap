// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InviteFriends2 InviteFriends2 component renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "paddingTop": 92,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingLeft": 53.99999999999999,
        }
      }
    >
      <View
        style={
          Object {
            "paddingTop": 9.857142857142856,
          }
        }
      >
        <ForwardRef
          onPress={[Function]}
        >
          <Image
            source={
              Object {
                "default": "",
              }
            }
            style={
              Object {
                "height": 22.17857142857143,
                "width": 36,
              }
            }
          />
        </ForwardRef>
      </View>
      <View
        style={
          Object {
            "paddingLeft": 166,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#1F1F60",
              "fontFamily": "BananaGrotesk-Bold",
              "fontSize": 32.85714285714286,
              "lineHeight": 44.35714285714286,
            }
          }
        >
          Invite Friends
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 50.92857142857143,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 54.214285714285715,
        }
      }
    />
    <View
      style={
        Object {
          "paddingLeft": 50,
        }
      }
    >
      <View
        style={
          Object {
            "width": 650,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#1F1F60",
              "fontFamily": "BananaGrotesk-Bold",
              "fontSize": 32.85714285714286,
              "lineHeight": 44.35714285714286,
            }
          }
        >
          Weâ€™ve found 
          118
           connections
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 50.92857142857143,
        }
      }
    />
    <View
      style={
        Object {
          "paddingLeft": 50,
        }
      }
    >
      <View
        style={
          Object {
            "width": 650,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View>
            <Text
              style={
                Object {
                  "color": "#1F1F60",
                  "fontFamily": "BananaGrotesk-Medium",
                  "fontSize": 29.571428571428573,
                  "lineHeight": 44.35714285714286,
                }
              }
            >
              Invite friends via email
            </Text>
          </View>
          <View
            style={
              Object {
                "justifyContent": "flex-end",
                "paddingBottom": 6.571428571428571,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#7B46E4",
                  "fontFamily": "BananaGrotesk-Bold",
                  "fontSize": 23,
                  "lineHeight": 26.285714285714285,
                }
              }
            >
              Invite All
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingTop": 50.92857142857143,
          }
        }
      />
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <View
          key="0"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "width": 210.00000000000003,
              }
            }
          >
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 46,
                }
              }
            >
              <Image
                key="0"
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "borderRadius": 49.285714285714285,
                    "height": 120,
                    "justifyContent": "center",
                    "width": 120,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "paddingTop": 18.071428571428573,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 40,
                }
              }
            >
              <View
                style={
                  Object {
                    "width": 132,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1F1F60",
                      "fontFamily": "BananaGrotesk-Light",
                      "fontSize": 23,
                      "lineHeight": 24.642857142857146,
                      "textAlign": "center",
                    }
                  }
                >
                  Amelia Jones
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 14.785714285714286,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 30,
                }
              }
            >
              <View
                style={
                  Object {
                    "borderColor": "#7B46E4",
                    "borderRadius": 26.285714285714285,
                    "borderWidth": 2,
                    "height": 49.285714285714285,
                    "width": 150,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "paddingTop": 8.214285714285715,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#7B46E4",
                        "fontFamily": "BananaGrotesk-Bold",
                        "fontSize": 23,
                        "lineHeight": 26.285714285714285,
                        "textAlign": "center",
                      }
                    }
                  >
                    Invite
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "paddingLeft": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "#E6ECED",
                  "borderWidth": 1,
                  "height": "100%",
                  "width": 2,
                }
              }
            />
          </View>
        </View>
        <View
          key="1"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "width": 210.00000000000003,
              }
            }
          >
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 46,
                }
              }
            >
              <Image
                key="1"
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "borderRadius": 49.285714285714285,
                    "height": 120,
                    "justifyContent": "center",
                    "width": 120,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "paddingTop": 18.071428571428573,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 40,
                }
              }
            >
              <View
                style={
                  Object {
                    "width": 132,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1F1F60",
                      "fontFamily": "BananaGrotesk-Light",
                      "fontSize": 23,
                      "lineHeight": 24.642857142857146,
                      "textAlign": "center",
                    }
                  }
                >
                  Amelia Jones
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 14.785714285714286,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 30,
                }
              }
            >
              <View
                style={
                  Object {
                    "borderColor": "#7B46E4",
                    "borderRadius": 26.285714285714285,
                    "borderWidth": 2,
                    "height": 49.285714285714285,
                    "width": 150,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "paddingTop": 8.214285714285715,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#7B46E4",
                        "fontFamily": "BananaGrotesk-Bold",
                        "fontSize": 23,
                        "lineHeight": 26.285714285714285,
                        "textAlign": "center",
                      }
                    }
                  >
                    Invite
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "paddingLeft": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "#E6ECED",
                  "borderWidth": 1,
                  "height": "100%",
                  "width": 2,
                }
              }
            />
          </View>
        </View>
        <View
          key="2"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "width": 210.00000000000003,
              }
            }
          >
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 46,
                }
              }
            >
              <Image
                key="2"
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "borderRadius": 49.285714285714285,
                    "height": 120,
                    "justifyContent": "center",
                    "width": 120,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "paddingTop": 18.071428571428573,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 40,
                }
              }
            >
              <View
                style={
                  Object {
                    "width": 132,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1F1F60",
                      "fontFamily": "BananaGrotesk-Light",
                      "fontSize": 23,
                      "lineHeight": 24.642857142857146,
                      "textAlign": "center",
                    }
                  }
                >
                  Amelia Jones
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 14.785714285714286,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 30,
                }
              }
            >
              <View
                style={
                  Object {
                    "borderColor": "#7B46E4",
                    "borderRadius": 26.285714285714285,
                    "borderWidth": 2,
                    "height": 49.285714285714285,
                    "width": 150,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "paddingTop": 8.214285714285715,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#7B46E4",
                        "fontFamily": "BananaGrotesk-Bold",
                        "fontSize": 23,
                        "lineHeight": 26.285714285714285,
                        "textAlign": "center",
                      }
                    }
                  >
                    Invite
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "paddingLeft": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "#E6ECED",
                  "borderWidth": 1,
                  "height": "100%",
                  "width": 2,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 83.78571428571429,
        }
      }
    />
    <View
      style={
        Object {
          "paddingLeft": 50,
        }
      }
    >
      <View
        style={
          Object {
            "width": 650,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View>
            <Text
              style={
                Object {
                  "color": "#1F1F60",
                  "fontFamily": "BananaGrotesk-Medium",
                  "fontSize": 29.571428571428573,
                  "lineHeight": 44.35714285714286,
                }
              }
            >
              Invite friends via contact
            </Text>
          </View>
          <View
            style={
              Object {
                "justifyContent": "flex-end",
                "paddingBottom": 6.571428571428571,
              }
            }
          >
            <ForwardRef
              onPress={[Function]}
            >
              <Text
                style={
                  Object {
                    "color": "#7B46E4",
                    "fontFamily": "BananaGrotesk-Bold",
                    "fontSize": 23,
                    "lineHeight": 26.285714285714285,
                  }
                }
              >
                Invite All
              </Text>
            </ForwardRef>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingTop": 50.92857142857143,
          }
        }
      />
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <View
          key="0"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "width": 210.00000000000003,
              }
            }
          >
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 46,
                }
              }
            >
              <Image
                key="0"
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "borderRadius": 49.285714285714285,
                    "height": 120,
                    "justifyContent": "center",
                    "width": 120,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "paddingTop": 18.071428571428573,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 40,
                }
              }
            >
              <View
                style={
                  Object {
                    "width": 132,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1F1F60",
                      "fontFamily": "BananaGrotesk-Light",
                      "fontSize": 23,
                      "lineHeight": 24.642857142857146,
                      "textAlign": "center",
                    }
                  }
                >
                  Name 1 
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 14.785714285714286,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 30,
                }
              }
            >
              <View
                style={
                  Object {
                    "borderColor": "#7B46E4",
                    "borderRadius": 26.285714285714285,
                    "borderWidth": 2,
                    "height": 49.285714285714285,
                    "width": 150,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "paddingTop": 8.214285714285715,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#7B46E4",
                        "fontFamily": "BananaGrotesk-Bold",
                        "fontSize": 23,
                        "lineHeight": 26.285714285714285,
                        "textAlign": "center",
                      }
                    }
                  >
                    Invite
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "paddingLeft": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "#E6ECED",
                  "borderWidth": 1,
                  "height": "100%",
                  "width": 2,
                }
              }
            />
          </View>
        </View>
        <View
          key="1"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "width": 210.00000000000003,
              }
            }
          >
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 46,
                }
              }
            >
              <Image
                key="1"
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "borderRadius": 49.285714285714285,
                    "height": 120,
                    "justifyContent": "center",
                    "width": 120,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "paddingTop": 18.071428571428573,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 40,
                }
              }
            >
              <View
                style={
                  Object {
                    "width": 132,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1F1F60",
                      "fontFamily": "BananaGrotesk-Light",
                      "fontSize": 23,
                      "lineHeight": 24.642857142857146,
                      "textAlign": "center",
                    }
                  }
                >
                  Name 2
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 14.785714285714286,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 30,
                }
              }
            >
              <View
                style={
                  Object {
                    "borderColor": "#7B46E4",
                    "borderRadius": 26.285714285714285,
                    "borderWidth": 2,
                    "height": 49.285714285714285,
                    "width": 150,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "paddingTop": 8.214285714285715,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#7B46E4",
                        "fontFamily": "BananaGrotesk-Bold",
                        "fontSize": 23,
                        "lineHeight": 26.285714285714285,
                        "textAlign": "center",
                      }
                    }
                  >
                    Invite
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "paddingLeft": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "#E6ECED",
                  "borderWidth": 1,
                  "height": "100%",
                  "width": 2,
                }
              }
            />
          </View>
        </View>
        <View
          key="2"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "width": 210.00000000000003,
              }
            }
          >
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 46,
                }
              }
            >
              <Image
                key="2"
                source={
                  Object {
                    "default": "",
                  }
                }
                style={
                  Object {
                    "borderRadius": 49.285714285714285,
                    "height": 120,
                    "justifyContent": "center",
                    "width": 120,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "paddingTop": 18.071428571428573,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 40,
                }
              }
            >
              <View
                style={
                  Object {
                    "width": 132,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#1F1F60",
                      "fontFamily": "BananaGrotesk-Light",
                      "fontSize": 23,
                      "lineHeight": 24.642857142857146,
                      "textAlign": "center",
                    }
                  }
                >
                  Name 3
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 14.785714285714286,
                }
              }
            />
            <View
              style={
                Object {
                  "paddingLeft": 30,
                }
              }
            >
              <View
                style={
                  Object {
                    "borderColor": "#7B46E4",
                    "borderRadius": 26.285714285714285,
                    "borderWidth": 2,
                    "height": 49.285714285714285,
                    "width": 150,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "paddingTop": 8.214285714285715,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#7B46E4",
                        "fontFamily": "BananaGrotesk-Bold",
                        "fontSize": 23,
                        "lineHeight": 26.285714285714285,
                        "textAlign": "center",
                      }
                    }
                  >
                    Invite
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "paddingTop": 16.42857142857143,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "paddingLeft": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "#E6ECED",
                  "borderWidth": 1,
                  "height": "100%",
                  "width": 2,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
